xarray/conventions.py

```diff
@@ -193,9 +193,15 @@ def decode_cf_variable(
         var = variables.Numpy2StringDTypeCoder().decode(var)
 
     if mask_and_scale:
+        dec_times = True if decode_times else False
+        dec_timedelta = True if decode_timedelta else False
```

Comment:
nit: is it possible to pass `decode_times` and `decode_timedelta` through to `CFMaskCoder` and `CFScaleOffsetCoder` without converting to `bool`, or perhaps do this conversion in the constructors?

Reply:
@spencerkclark Yes, it is possible. I had to refactor out some code into a new `common.py` to prevent circular imports.

Reply:
Nice, thanks!